### DISCUSSION TITLE
AskVSUS not working properly...

### DISCUSSION THREAD

**Discussion by SusheelThapa** [Rajendrakhanal/vsus#33] [Q&A] [answered]
When i ask question to the Ask VSUS it doesn't give me answer...

![AskVSUS not working](https://user-images.githubusercontent.com/83917129/216363490-d404413d-d3d6-4b6f-b52d-13eb14400b51.png)

**Comment by Rajendrakhanal** [comment, marked as answer]
As ASKVSUS uses OPENAI API so,
First you should setup for up OpenAI Key 

1.Login/Signup to you OPENAI account
2.Create a new secret key
3.Copy the newly generated secret key.
4.Open your vscode setting.
5.Search for apikey
6.In the input box, paste the OpenAI key.

After this Setup then ASKVSUS will work properly.

**Reply by SusheelThapa** [reply]
Thanks @Rajendrakhanal  It worked...

![Async Worked](https://user-images.githubusercontent.com/83917129/216367302-92773197-d04a-4b1b-9c32-6d64dd0700bc.png)
